spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/config/LoadBalancerProperties.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gateway.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Ryan Baxter
+ */
+@ConfigurationProperties("spring.cloud.gateway.loadbalancer")
```

Comment:
This looks unrelated and unused. Please remove.

Reply:
yes,i will remove this change.